ksso/failure_access_prohibited_message.html

```diff
@@ -0,0 +1,34 @@
+<html>
+  <head>
+    <title>Authentication Successful</title>
```

Comment:
Auth shouldn't be successful here, right?

Reply:
Glaz almaz :)))